### PR TITLE
Tutorial for "Enemy Location"『エネミーの出現予測位置』

### PR DESCRIPTION
**Pull request by SanseiMuramasa**

### CODE REVIEW

File: Files/co_040017.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Title#0,"""『エネミーの出現予測位置』について"""
-text01#0,"""『エネミーの出現予測位置』が<br>レーダーマップ上に黄色のサークルで<br>表示されます。<br><br>ダッシュパネルで高速移動して<br>この出現予測位置にすばやく向かい<br>出現したエネミーを討伐しましょう。<br><br>すべて討伐すると、<br>次の出現予測位置が表示されます。"""
+Title#0,"""Enemy Location"""
+text01#0,"""The yellow circle on the radar map shows the enemies' location.<br><br>Use the dash panels to boost yourself<br>quickly toward's the enemies' location.<br>From that location, you can encounter the enemy.<br><br>Once you have defeated all the enemies in that location, the next enemy location will appear on the map."""
```
Comment (SynthSy) [inline Files/co_040017.csv]: towards, not toward's

**Comment by Aida-Enna** [comment]
The PR is failing because you need to move the translated file out of the Files folder and into the translated folder based on the file's contents.

**Comment by SanseiMuramasa** [comment]
How do I do that?

**Comment by alama** [comment]
https://blog.github.com/2013-03-15-moving-and-renaming-files-on-github/

**Comment by SanseiMuramasa** [comment]
Where should I move the file?

**Comment by Aida-Enna** [comment]
Like Alam posted above, if you're using Github's online editor, hit backspace to delete the "/Files/" and type wherever it should go, like this: https://gfycat.com/AccurateJadedAsiaticmouflon

**Pull request merged**